network-api/networkapi/wagtailpages/models.py

```diff
@@ -14,9 +14,13 @@
     PrimaryPage,
     Styleguide,
 )
+
+# Blog models:
```

Comment:
Comments like this are a bit unscary, devs should be able to recognise what models they are from reading the import

Reply:
This was more about readability, to separate each section, but it can be removed